tests/e2e/provisioning/test/e2e_provisioning_test.go

```diff
@@ -46,4 +49,33 @@ func Test_E2E_Provisioning(t *testing.T) {
 
 	err = ts.dashboardChecker.AssertRedirectedToUAA(dashboardURL)
 	assert.NoError(t, err)
+
+	if ts.IsTestAzureEventHubsEnabled {
+		checkAzureEventHubProperties(ts, t, operationID)
+	}
+}
+
+func checkAzureEventHubProperties(ts *Suite, t *testing.T, operationID string) {
+	resourceGroup, err := (*ts.azureClient).GetResourceGroup(context.TODO(), ts.brokerClient.GetClusterName())
+	assert.NoError(t, err)
+	assert.Equal(t, http.StatusOK, resourceGroup.StatusCode, "HTTP GET fails for ResourceGroup")
+	assert.NotNil(t, resourceGroup, "ResourceGroup name is nil")
```

Comment:
its a struct, not a pointer to a struct, so it can never be nil, and your not checking the name here

Reply:
This is changed to listing of resourcegroups hence this is not applicable anymore.

Name will be a generated one by KEB hence not checking it at all(in the new code).